client/blocks/reader-featured-video/index.jsx

```diff
@@ -55,6 +62,12 @@ class ReaderFeaturedVideo extends React.Component {
 
 	handleThumbnailClick = ( e ) => {
 		e.preventDefault();
+		this.props.onThumbnailClick();
```

Comment:
hmmm... if allowPlaying is set to false, should we just let the click propagate? just return first thing?